.pre-commit-config.yaml

```diff
@@ -13,6 +16,10 @@ repos:
   hooks:
   - id: gdformat
   - id: gdlint
+- repo: https://github.com/fsfe/reuse-tool
+  rev: v5.0.2
+  hooks:
+  - id: reuse
```

Comment:
My spider sense tells me that this might be incredibly annoying during development...

Reply:
Another option would be to have a separate CI job that checks this – less annoying during development but would still catch errors in CI, and also makes the "Linting and Formatting" check name not be inaccurate – it would be clearer if a "License information" check failed in the case where something is broken.

Reply:
I think that's better. We even get a nice summary on the workflow page almost for free:

https://github.com/endlessm/threadbare/actions/runs/14201291432?pr=88

![image](https://github.com/user-attachments/assets/056ee96c-08d7-4087-913d-97e278fecb60)